src/distribute_kv_cache/tests/kvcache.rs

```diff
@@ -0,0 +1,176 @@
+use std::sync::Once;
+
+use tracing::level_filters::LevelFilter;
+use tracing_subscriber::{
+    filter, fmt::layer, layer::SubscriberExt, util::SubscriberInitExt, Layer,
+};
+
+/// Use for unit test to setup tracing
+#[allow(dead_code)]
```

Comment:
why dead code?

Reply:
Previously, there were some tools and methods that allowed lowering the level in tests. However, it is now possible to consider removing dead code.